src/sst/elements/vanadis/vanadis.cc

```diff
@@ -1378,7 +1388,7 @@ VANADIS_COMPONENT::tick(SST::Cycle_t cycle)
     if(output_verbosity >= 2)
```

Comment:
The output level shouldn't match this? Is output_verbosity not being passed as part of output (e.g. why this check and then the output)?

Reply:
It should match, fixing that. Usually the check is to avoid having to call verbose() in the first place which has non-negligible overhead when you do it every cycle. We should eventually ifdef and/or convert these to output->debug() to eliminate the overhead if you're not printing anything.